starport/templates/module/create/oracle/x/{{moduleName}}/client/cli/tx_oracle.go.plush

```diff
@@ -0,0 +1,147 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/spf13/cobra"
+	"<%= modulePath %>/x/<%= moduleName %>/types"
+)
+
+var _ = strconv.Itoa(0)
```

Comment:
FYI this line is used in some templates because `strconv` package is only used in some context (if there is a int to read for CLI) and to ensure the import statement never return an error

In this case `strconv` is always used as I can see below so this line can be removed